src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
In `ColorSupplier` you set the new line after the class name, yet you didn't do that here.
Please keep the code consistent - in my opinion, you're better off without unnecessary new lines.